docs/blog/Webots-2021-a-release.md

```diff
@@ -0,0 +1,93 @@
+# Version R2021a Released
+
+<p id="publish-data">By Darko Lukic - **XX**th December 2020</p>
+
+---
+
+Webots R2021a is there! And it's packed with some new features, improvements and, of course, bug fixes.
+
+However, this article only lists some of the major changes.
+Please refer to the [R2021 changelog](../reference/changelog-r2021.md) for a comprehensive list of all changes.
+
+
+## Stars ⭐️ and Milestones 🎉
+
+Webots GitHub repository has reached more than 1000 stars!
+This is fantastic news for us as it shows that the Webots community has been growing ahead of our expectations.
+It has been 2 years since we started the exciting open-source journey.
+We are thankful to all contributors and partners who made the Webots open-source dream becomes a reality.
+
+This year will be also remembered by other milestones:
+- Our Discord community is now counting over 1000 members.
+- The YouTube Cyberobotics channel has exceeded 1000 subscribers.
+- The recently released Webots Docker container has yielded almost 5000 downloads.
+
+----
+
+## Camera Image Segmentation
+
+Webots now is ready for your next deep learning project as it now supports [object recognition and segmentation](https://cyberbotics.com/doc/reference/camera#wb_camera_has_recognition)!
+
+%figure "Image Segmentation"
+![Image Segmentation](images/image_segmentation.gif)
+%end
+
+This new functionality reuses the existing camera recognition functionality to classify the objects in the scene based on the properties of the object instance.
+This generic and customizable method allows the user to apply any type of segmentation: semantic, instance, or panoptic seg-mentation.
+Simply setting the recognition property at the object model layer, will generate a semantic segmentation.
+
+---
+
+## ROS 2 & Webots
+
+We are continuing to develop and improve the support for ROS 2 in Webots with now [13 Webots packages for ROS 2](https://github.com/cyberbotics/webots\_ros2).
```

Comment:
What is the correct spelling `ROS2` or `ROS 2`?

Reply:
I would go for `ROS2`.

Reply:
On a second thought, I believe "ROS 2" is the official spelling see https://index.ros.org/doc/ros2/. I will revert my changes.